packages/web-react/src/components/Toast/ToastBarMessage.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { ChildrenProps } from '../../types';
+
+const ToastBarMessage = ({ children }: ChildrenProps) => {
+  return <div>{children}</div>;
```

Comment:
please set the truncate classes

Reply:
and we need unsafe props, because then the user can change CSS variable to how many lines he use.